rsmtool/utils.py

```diff
@@ -61,6 +63,7 @@
             'use_scaled_predictions_new': False,
             'select_transformations': False,
             'standardize_features': True,
+            'images_to_thumbnails': False,
```

Comment:
Why not just `use_thumbnails`? For boolean variables, it's usually best to have a verb in there just like we do for most of the others.